assets/agenda/utils.ts

```diff
@@ -1041,12 +1042,13 @@ export const getCoverageTooltip = (coverage: any, beingUpdated?: any) => {
             assignedDetails,
         });
     } else if (coverage.workflow_status === WORKFLOW_STATUS.ASSIGNED) {
-        return gettext('Planned {{ type }} coverage {{ slugline }}, expected {{date}} at {{time}} {{assignedDetails}}', {
+        return gettext('{{ status }} {{ type }} coverage {{ slugline }}, expected {{date}} at {{time}} {{assignedDetails}}', {
```

Comment:
I don't believe this is going to make much sense using the `getCoverageStatusText` here.

For example, a Planned coverage currently states:
`Planning text coverage {{ slugline }}, expected at ....`

After these changes it would state something like:
`Coverage planned text coverage {{ slugline }}, expected at ....` which has the repeating `coverage`